.github/workflows/push_images.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'feature/official_steamcmd'
+
+jobs:
+  docker:
+    strategy:
+      matrix:
+        include:
+           - imagename: Dockerfile.vanilla
+             tagname: vanilla
+           - imagename: Dockerfile.wine6
+             tagname: wine6
+           - imagename: Dockerfile.wine7
+             tagname: wine7
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          file: ${{ matrix.imagename }}
+          push: true
+          tags: avivace/ror2server:${{ matrix.tagname }}
```

Comment:
Relating to my other comment:

> I argue we should add a static part (a version number, a codename, ...) to each `tagname` so we can push non-breaking updates to the same tag and breaking updates to a new tag. I'll try to come up with something we can discuss about.

Imajine we use `v0.7-` (since our last released version was `v0.6`) as the static part.
Moreover, in this suggestion I propose a way to introduce the latest tag.

```suggestion

jobs:
  docker:
    strategy:
      matrix:
        include:
           - imagename: Dockerfile.vanilla
             tagnames: avivace/ror2server:v0.7-vanilla,avivace/ror2server:latest
           - imagename: Dockerfile.wine6
             tagnames: avivace/ror2server:v0.7-wine6
           - imagename: Dockerfile.wine7
             tagnames: avivace/ror2server:v0.7-wine7
    runs-on: ubuntu-latest
    steps:
      -
        name: Set up QEMU
        uses: docker/setup-qemu-action@v1
      -
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1
      -
        name: Login to DockerHub
        uses: docker/login-action@v1 
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      -
        name: Build and push
        uses: docker/build-push-action@v2
        with:
          file: ${{ matrix.imagename }}
          push: true
          tags: ${{ matrix.tagnames }}
```

_(Note this is an example and I'm not a GitHub Actions expert by any means)_

Reply:
It generates one case, but `tags` [is documented as being a CSV](https://github.com/docker/build-push-action#inputs).

Reply:
Marking it as resolved because we won't introduce a static part to the tags.
The proposal still stands for `tags: avivace/ror2server:vanilla,avivace/ror2server:latest`.